test/support/interfaces/put-facility-by-facility-identifier-acbs-endpoints.interface..ts

```diff
@@ -0,0 +1,6 @@
+import nock from 'nock/types';
+
+export interface PutFacilityAcbsEndpoints {
```

Comment:
I would change this to `PutFacilityAcbsRequests` (aren't they requests, rather than endpoints?)